blueprints/load-scheduling/java-gc-generic/utils.libsonnet

```diff
@@ -0,0 +1,31 @@
+function(config, agent_group='default') {
+  // Define the target address directly
+  local jmx_target_address = config.policy.jmx.jmx_host + ':' + config.policy.jmx.jmx_prometheus_port,
+
+  jmx_inframeter: {
+    agent_group: agent_group,
+    per_agent_group: true,
```

Comment:
Per agent group should be false for jmx blueprint so that each agent can collect metrics from service instance running next to it

Reply:
Can also, make this parameter configurable because it's highly dependent on the user's deployment